roxctl/netpol/connectivity/diff/netpol_test.go

```diff
@@ -0,0 +1,285 @@
+package diff
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/roxctl/common/environment/mocks"
+	"github.com/stackrox/rox/roxctl/common/npg"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestDiffAnalyzeNetpolCommand(t *testing.T) {
+	t.Parallel()
+	suite.Run(t, new(diffAnalyzeNetpolTestSuite))
+}
+
+type diffAnalyzeNetpolTestSuite struct {
+	suite.Suite
+}
+
+func (d *diffAnalyzeNetpolTestSuite) TestValidDiffCommand() {
+	cases := []struct {
+		name                  string
+		inputFolderPath1      string
+		inputFolderPath2      string
+		outFile               string
```

Comment:
This is used only once and has the same value in this test. Could we remove it from the params and have a constant/var instead.

Reply:
sure will add empty values in the other tests , and a new test with `non-existing` file which expects no error

Reply:
done